cmake/Dependencies.cmake

```diff
@@ -85,6 +85,8 @@ if(NOT BUILD_ATEN_MOBILE)
   set(BLAS "MKL" CACHE STRING "Selected BLAS library")
 else()
   set(BLAS "Eigen" CACHE STRING "Selected BLAS library")
+  set(AT_MKLDNN_ENABLED 0)
```

Comment:
Shouldn't these be ignored for BLAS "Eigen"? If not, that's probably the better fix to put somewhere.

Reply:
I think you are right - we shouldn't need explicit set these to 0. The places that set these to 1 below are already covered by "NOT BUILD_ATEN_MOBILE" condition.
@t-vi - any particular reason to set them here?

Reply:
@orionr looks like these are needed by Config.h.in - we need set some default value otherwise it will cause compile error. Config.h is used by Aten code which was not compiled for mobile (BUILD_ATEN_MOBILE) previously. Now we need set them.

Reply:
I'm a lazy person, so most of the things I put in were needed at some point of time. :slightly_smiling_face:

Reply:
Sounds reasonable. This could use some cleanup later, but good for now.